src/theme.js

```diff
@@ -206,7 +207,7 @@ const base = {
     },
     progress: {
       bg: '#ebebeb',
-      color: '#4ab57f',
+      color: 'purple.0',
```

Comment:
Should this not be uni purple #6b2c90 ?

Reply:
purple.0 is defined as #6b2c90 on line 55

Reply:
Couldn't see it. I searched for purple.0 Duh!